angular-locale_en-cn.js

```diff
@@ -80,10 +80,10 @@ angular.module("ngLocale", [], ["$provide", function($provide) {
                 5,
                 6
             ],
-            "fullDate": "y\u5e74M\u6708d\u65e5EEEE",
-            "longDate": "y\u5e74M\u6708d\u65e5",
-            "medium": "y\u5e74M\u6708d\u65e5 ah:mm:ss",
-            "mediumDate": "y\u5e74M\u6708d\u65e5",
+            "fullDate": "y, M d, EEEE",
+            "longDate": "y, M d ",
+            "medium": "y, M d ah:mm:ss",
+            "mediumDate": "y, M d ",
```

Comment:
There are still commas after the y that should not be there. Once those are removed I will merge :)